euth_wagtail/assets/scss/components/_dashboard-form-buttons.scss

```diff
@@ -0,0 +1,52 @@
+#dashboard-form-buttons {
```

Comment:
Ok, afaik browsers map CSS ids and URL. So requesting `/en/dashboard/projects/title-of-project/#dashboard-form-buttons` results in jumping to the corresponding buttons. This might be a con.